src/reactapp/src/utils/address.js

```diff
@@ -72,6 +73,27 @@ const addressInitValues = {
   country: '',
 };
 
+const requiredMessage = __('%1 is required');
+
+export const addressInitialValidationSchema = {
```

Comment:
@rajeev-k-tomy what if people customize one of the two addresses in their Magento project? (e.g. required company/VAT in billing address, but not shipping). How would they now achieve that?

do they need to extend the `validationSchema` object?

Like this?
```
const validationSchema = useRegionValidation(
    selectedCountry,
    addressInitialValidationSchema,
    { vat_number: YupString().required("VAT is required") }
  );
```

Is that ideal? If so, should it be documented maybe?

Reply:
Good question. Yes, this is currently make a common initial validation schema for both billing and shipping. I will define a separate initial validation schema for both billing and shipping addresses so that someone can modify them directly inside form manager component.

Reply:
I improved the logic. Please  have a look again

Reply:
great! thank you Rajeev! 🙏